XRTK-Core/Assets/XRTK.Tests/Services/ITestExtensionServiceProvider1.cs

```diff
@@ -3,5 +3,5 @@
 
 namespace XRTK.Tests.Services
 {
-    internal interface ITestExtensionServiceProvider1 : ITestDataProvider1, ITestExtensionService1 { }
+    internal interface ITestExtensionServiceProvider1 : ITestDataProvider1 { }
```

Comment:
This should probably be derived from `ITestServiceProvider1` not a data provider